core/standard/clause_7_building_blocks.adoc

```diff
@@ -82,7 +82,7 @@ include::recommendations/record-core/PER_additional-properties-record.adoc[]
 [[sc_record_extensions]]
 ===== Record extensions
 
-In tables <<core-properties-record-table>> and <<core-properties-resource-table>> this standard defines a set of core properties for describing discoverable resources.  It is anticipated that this set of core properties will be extended to suite specific use cases or requirements.  For example, a catalog record may be extended to include additional properties from the https://semiceu.github.io/GeoDCAT-AP/releases/[GeoDCAT] vocabulary.  The fact that a catalog record has been extended in this way can be advertised using the `conformsTo` member.  The `conformsTo` member is a list of identifiers that indicate each extension used in a record.  This specification does not define the specific identifiers that are used to identify specific extensions; it simply provides a place where these identifiers can be listed.
+In <<core-properties-record-table>> and <<core-properties-resource-table>> this Standard defines a set of core properties for describing discoverable resources.  It is anticipated that this set of core properties will be extended to suite specific use cases or requirements.  For example, a catalog record may be extended to include additional properties from the https://semiceu.github.io/GeoDCAT-AP/releases/[GeoDCAT] vocabulary.  The fact that a catalog record has been extended in this way can be advertised using the `conformsTo` member.  The `conformsTo` member is a list of identifiers that indicate each extension used in a record.  This Specification does not define the specific identifiers that are used to identify specific extensions; it simply provides a place where these identifiers can be listed.
```

Comment:
Replace Specification to Standard?

Reply:
@kalxas OGC Policy is that OGC documents that specify requirements/conformance and are approved by the OGC Membership are called Standards (with a capital "S".  Used to be called Implementation Specifications but in the mid 2000s the OAB approved and Members agreed to a policy change to call these documents Standards.